src/routes.ts

```diff
@@ -1,16 +1,21 @@
-import { StackNavigator } from 'react-navigation'
+import { StackNavigator, } from 'react-navigation'
 import { Home } from 'src/ui/home/'
-import { SeedPhrase } from 'src/ui/registration/containers/seedPhrase'
+import { Landing } from 'src/ui/landing/'
+import { PasswordEntry, SeedPhrase } from 'src/ui/registration'
 
-export const RootStack = StackNavigator({
-    Home: {
-      screen: Home
-    },
-    SeedPhrase: {
-      screen: SeedPhrase
-    }
+export const enum routeList {
+  Landing = 'Landing',
+  PasswordEntry = 'PasswordEntry',
+  SeedPhrase = 'SeedPhrase',
+  Home = 'Home'
+}
+
+export const Routes = StackNavigator({
+    Landing: { screen: Landing },
+    PasswordEntry: { screen: PasswordEntry },
+    SeedPhrase: { screen: SeedPhrase },
+    Home: { screen: Home }
   }, {
-    initialRouteName: 'SeedPhrase',
     headerMode: 'none'
```

Comment:
may need to reconsider this way of getting rid of nav bar if we need end up needing it on the dashboard.

Reply:
Thank you, addressed.